client/my-sites/sidebar-unified/style.scss

```diff
@@ -14,13 +14,15 @@
 	--color-sidebar-menu-hover-heading-background: #1a1e23;
 	--color-sidebar-menu-hover: #00b9eb;
 	--color-sidebar-menu-hover-text: #00b9eb;
-	--color-sidebar-menu-selected-background: #0073aa;
+	--color-sidebar-menu-selected-background: #2271b1;
```

Comment:
Should we be using color studio variables here i.e. --studio-blue-50?

Reply:
This is a prototype. :)